content/en/tests/setup/dotnet.md

```diff
@@ -389,11 +389,11 @@ The test session name needs to be unique within a repository to help you disting
 
 #### When to use `DD_TEST_SESSION_NAME`
 
-There's a set of parameters that the product checks to establish correspondence between test sessions. The test command used to execute the tests is one of them. If the test command contains a string that changes for every execution, such as a temporary folder, Datadog considers the sessions to be unrelated to each other. Some examples of unstable test commands are:
+There's a set of parameters that Datadog checks to establish correspondence between test sessions. The test command used to execute the tests is one of them. If the test command contains a string that changes for every execution, such as a temporary folder, Datadog considers the sessions to be unrelated to each other. For example:
 
-- `DD_TEST_SESSION_NAME=integration-tests dotnet test --temp-dir=/var/folders/t1/rs2htfh55mz9px2j4prmpg_c0000gq/T`
+- `dotnet test --temp-dir=/var/folders/t1/rs2htfh55mz9px2j4prmpg_c0000gq/T`
```

Comment:
@juan-fernandez, just checking if we should change this example as shown. This would match the examples for the other languages, which include just the test command (and omit the variable assignment part).

Reply:
Alternatively, perhaps add a second example with the env variable included below the guidance about where it's useful?

Reply:
yeah, that looks correct, thanks